packages/graphql-server/src/resolvers/appointment/__mocks__/appointment.ts

```diff
@@ -0,0 +1,68 @@
+import {AppointmentModel} from '../../../types'
```

Comment:
The branch needs linting - I can see there aren't the correct spacings in the files